common/src/main/java/org/opensearch/ml/common/output/model/ModelTensors.java

```diff
@@ -35,6 +35,12 @@ public ModelTensors(List<ModelTensor> mlModelTensors) {
         this.mlModelTensors = mlModelTensors;
     }
 
+    @Builder
```

Comment:
Can we have two `@Builder` annotations on different constructors in the same class with Lombok?